src/components/BadgeSlider/index.tsx

```diff
@@ -11,16 +21,98 @@ export interface IBadge {
   title: string;
 }
 
+const ITEMS_PER_PAGE = 3;
+
 interface Props {
   items: IBadge[];
 }
 
 const BadgeSlider: React.FC<Props> = ({ items }) => {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const totalPages = Math.ceil(items.length / ITEMS_PER_PAGE);
+  const slides = chunk(items, ITEMS_PER_PAGE);
+  const hideControls = items.length === 0 || items.length === ITEMS_PER_PAGE;
+
   return (
-    <>
-      <DesktopSlider items={items} />
-      <MobileSlider items={items} />
-    </>
+    <CarouselProvider
+      naturalSlideWidth={500}
+      naturalSlideHeight={200}
+      isIntrinsicHeight
+      totalSlides={totalPages}
+      currentSlide={currentSlide}
+      className="relative max-w-8xl mx-auto"
+    >
+      <Slider>
+        {slides.map((slide, index) => (
+          <Slide
+            key={index}
+            index={index}
+            innerClassName="grid grid-cols-1 gap-12 justify-items-center lg:px-20 lg:grid-cols-3"
+          >
+            {slide.map((badge: IBadge) => (
+              <Badge
+                key={badge.title}
+                imageSrc={badge.src}
+                imageAlt={badge.alt}
+                title={badge.title}
+                backgroundColor={badge.backgroundColor}
+                text={badge.text}
+                className="mx-auto"
+              />
+            ))}
+          </Slide>
+        ))}
+      </Slider>
+      {!hideControls && (
+        <>
+          <ButtonBack
+            className={classnames(
+              'hidden lg:block absolute left-0 top-0 h-full'
+            )}
+            onClick={() => setCurrentSlide((slide) => slide - 1)}
+          >
+            <span
+              className="material-icons text-gray"
+              style={{
+                fontSize: '5rem'
+              }}
+            >
+              navigate_before
+            </span>
+          </ButtonBack>
+          <ButtonNext
+            className={classnames(
+              'hidden lg:block absolute right-0 top-0 h-full'
+            )}
+            onClick={() => setCurrentSlide((slide) => slide + 1)}
+          >
+            <span
+              className="material-icons text-gray"
+              style={{
+                fontSize: '5rem'
```

Comment:
Shouldn't we use tailwind for this? If this is used here I bet it will be used in other areas of the page. Take a look at https://tailwindcss.com/docs/font-size where they add a new font size `'7xl': '5rem'` so then we can do `text-7xl`

Reply:
[You actually already did it, so you know how/what to do](https://github.com/debtcollective/home/blob/9d7b7139b1dc44570d888853d2957728367f0da4/tailwind.config.js#L40)

Reply:
Yes, indeed. But for some reason the styles were not being applied by using tailwind :(